features/bevragen/autorisatie-gba.feature

```diff
@@ -0,0 +1,590 @@
+      | code     | authorization                                                           |
+      | instance | /haalcentraal/api/brp/personen                                          |
+
+    @fout-case
+    Abstract Scenario: Afnemer vraagt om gegevensgroep <fields> en is niet geautoriseerd voor <missende autorisatie>
+      Gegeven de afnemer met indicatie '12345' heeft de volgende 'autorisatie' gegevens
+      | Rubrieknummer ad hoc (35.95.60) | Medium ad hoc (35.95.67) | Datum ingang (35.99.98) |
+      | <ad hoc rubrieken>              | N                        | 20201128                |
+      En de geauthenticeerde consumer heeft de volgende 'claim' gegevens
+      | naam         | waarde |
+      | afnemerID    | 12345  |
+      | gemeenteCode |        |
+      En de persoon met burgerservicenummer '000000024' heeft de volgende gegevens
+      | geboortedatum (03.10) | geslachtsnaam (02.40) | voornamen (02.10) | geslachtsaanduiding (04.10) | aanduiding naamgebruik (61.10) |
+      | 19830526              | Maassen               | Pieter            | M                           | E                              |
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000024                       |
+      | fields              | <fields>                        |
+      Dan heeft de response een object met de volgende gegevens
+      | naam     | waarde                                                                  |
+      | type     | https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.3             |
+      | title    | U bent niet geautoriseerd voor één of meerdere opgegeven field waarden. |
+      | status   | 403                                                                     |
+      | detail   | De foutieve fields waarden zijn: fields[0].                             |
+      | code     | authorization                                                           |
+      | instance | /haalcentraal/api/brp/personen                                          |
+
+      Voorbeelden:
+      | fields          | ad hoc rubrieken              | missende autorisatie                         |
+      | naam            | 10120 10210 10220 10230 10240 | aanduidingNaamgebruik (16110)                |
+      | geboorte        | 10120 10310                   | geboorteplaats (10320), geboorteland (10330) |
+      | nationaliteiten | 10120 40510 46310 46410 46510 | datumIngangGeldigheid (48510)                |
+
+
+  Rule: Vragen met fields gevraagd om een veld dat is afgeleid van één of meerdere andere velden, wanneer de gebruiker niet geautoriseerd is voor al die velden, geeft een foutmelding
+
+    @fout-case
+    Scenario: Afnemer vraagt om afgeleid gegeven <fields> en is niet geautoriseerd <missende autorisatie>
```

Comment:
`... en is niet geautoriseerd <missende autorisatie>` --> `... en is niet geautoriseerd voor <missende autorisatie>`